ykrt/build.rs

```diff
@@ -7,4 +7,7 @@ pub fn main() {
     println!("cargo:rustc-cfg=jitc_yk");
     // Always compile in the HWT tracer.
     println!("cargo:rustc-cfg=tracer_hwt");
+    // FIXME: This is a temporary hack because LLVM has problems if the main thread exits before
+    // compilation threads have finished.
```

Comment:
Do we have an issue we can link? If so, is the issue up to date with our current findings?

Reply:
I guess https://github.com/ykjit/yk/issues/841 is an example, but probably not the only one. @Pavel-Durov ?

Reply:
Maybe don't worry about it.